.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: Build Book
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+      - run: bundle install
+      - run: rake build:CI
+      - uses: actions/github-script@v2
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const fs = require('fs').promises;
+
+            const { repo: { owner, repo }, sha } = context;
+
+            const release = await github.repos.createRelease({
+              owner, repo,
+              tag_name: process.env.GITHUB_REF,
+              draft: true,
+              target_commitish: sha
+            });
+
+            console.log('A release has been created');
+            const build_dir = 'build';
+
+            for (let file of await fs.readdir(build_dir)) {
+              if (file == '.gitkeep') continue;
+              console.log('uploading', file);
+              await github.repos.uploadReleaseAsset({
```

Comment:
I don't see attached file on [your release](https://github.com/kerolloz/api_on_rails/releases/tag/226225735). Are you sure this works ?

Reply:
Oh sorry. Sure it does work. I think that is because it is a _draft_ release. 
![image](https://user-images.githubusercontent.com/36763164/91636469-1a941280-ea01-11ea-8d91-09f60a506ccc.png)